modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/1-mdf-claim.json

```diff
@@ -173,9 +173,6 @@
 									{
 										"key": "approved"
 									},
```

Comment:
I think this wouldn't need to be here as we talked about. This can also be removed